easybuild/easyconfigs/s/seq2HLA/seq2HLA-20180802-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'Tarball'
+
+name = 'seq2HLA'
+version = '20180802'
+commit = 'a4d26bd'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/TRON-Bioinformatics/seq2HLA'
+description = """ In-silico method written in Python and R to determine HLA genotypes of a sample.
+ seq2HLA takes standard RNA-Seq sequence reads in fastq format as input,
+ uses a bowtie index comprising all HLA alleles and outputs the most likely
+ HLA class I and class II genotypes (in 4 digit resolution),
+ a p-value for each call, and the expression of each class. """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/TRON-Bioinformatics/seq2HLA/archive']
+sources = [{'download_filename': '%s.tar.gz' % commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['50bc6242b90be19b8b3c1013b200de408cb9f0b142c7cd5783eb33eae91f94ba']
+
+dependencies = [
+    ('Biopython', '1.72', versionsuffix),
+    ('Bowtie', '1.2.2'),
+    ('Python', '2.7.15'),
+    ('R', '3.5.1', versionsuffix),
+]
+
+postinstallcmds = [
+    'sed -i -e "1i #!/usr/bin/env python" %(installdir)s/seq2HLA.py',
```

Comment:
@smoors You should also enable execution permissions (to avoid that users have to use `python $EBROOTSEQ2HLA/seq2HLA.py` and can just run `seq2HLA.py`)?

Reply:
it already has execute permissions, so this is not necessary.
I guess your plan is to go further with my PR? I will add your sanity check command then.

Reply:
#6965 already got merged, but it indeed makes sense to go forward with both because of the different toolchains.

Do you mind syncing this PR with `develop`, and adding the missing `R` dependency in the `seq2HLA` easyconfig that got merged in #6965?

I guess you can drop the post-install command I added for the permissions too since they're fine already.

Reply:
sure, but shouldn't you create a Python2 version of R with intel/2018a first?

Reply:
@smoors After thinking about this a bit, I'm not sure it's worth that effort...

Let's go forward in this PR with including `R` as dependency for `seq2HLA`, but not for older toolchains. As far as I understand, `R` is very optional anyway as dependency here?

Reply:
I don't think R is optional actually. but that should not hold this PR back, right? :)